content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -224,6 +222,34 @@ It is recommended that you disable this plugin in the scheduling profile when
 using default constraints for `PodTopologySpread`.
 {{< /note >}}
 
+#### A kubernetes default
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+When the feature gate `DefaultPodTopologySpreadPlugin` is enabled, the following
+default constraints for `PodTopologySpreadPlugin` are applied:
+
+```yaml
+defaultConstraints:
+  - maxSkew: 3
+    topologyKey: "kubernetes.io/hostname"
+    whenUnsatisfiable: ScheduleAnyway
+  - maxSkew: 5
+    topologyKey: "topology.kubernetes.io/zone"
+    whenUnsatisfiable: ScheduleAnyway
+```
+
+Also, the legacy `SelectorSpread` plugin is disabled.
+
+{{< note >}}
+If your nodes are not expected to have **both** `kubernetes.io/hostname` and
+`topology.kubernetes.io/zone` we recommend that you define your own constraints
```

Comment:
_nit_, doesn't block a merge
To [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we), how about:
```suggestion
`topology.kubernetes.io/zone` keys set, you should define your own constraints
```
?